test/ibm/runtime/test_runtime.py

```diff
@@ -71,16 +71,52 @@ class TestRuntime(IBMTestCase):
         "description": "Test program.",
         "max_execution_time": 300,
         "version": "0.1",
-        "backend_requirements": {"min_num_qubits":  5},
-        "parameters": [
-            {'name': 'param1', 'description': 'Desc 1', 'type': 'str', 'required': True},
-            {'name': 'param2', 'description': 'Desc 2', 'type': 'int', 'required': False}],
-        "return_values": [
-            {"name": "ret_val", "description": "Some return value.", "type": "string"}
-        ],
-        "interim_results": [
-            {"name": "int_res", "description": "Some interim result", "type": "string"},
-        ]
+        "backend_requirements": {
+            "min_num_qubits":  5
+        },
+        "parameters": {
+            "type": "object",
```

Comment:
I don't think that's the new metadata format. It's more like
```
{
  "id": "sample-program-abcdef123456",
  "name": "sample-program",
  "cost": 18000,
  "description": "A sample runtime program",
  "creation_date": "2021-05-10T08:59:51Z",
  "update_date": "2021-06-04T14:11:19Z",
  "spec": {
    "backend_requirements": {
      "min_num_qubits": 5
    },
    "parameters": {
      "properties": {
        "iterations": {
          "description": "Number of iterations to run. Each iteration generates a runs a random circuit.",
          "type": "integer",
          "minimum": 0
        }
      },
      "$schema": "https://json-schema.org/draft/2019-09/schema",
      "required": [
        "iterations"
      ]...
``` 

See https://runtime-us-east-dev.quantum-computing.ibm.com/openapi/#/Programs/get_program